data/seeds/games.js

```diff
@@ -0,0 +1,24 @@
+exports.seed = function (knex, Promise) {
+  return knex('games').del()
```

Comment:
Usually `truncate` is better to use than `del` since it also resets auto-incrementing tables. If you were to use `truncate` here, you wouldn't need to specify the ids in the seed data (as it'd always be 1, 2, 3 by default)